pkg/datapath/linux/ipsec/ipsec_linux.go

```diff
@@ -368,6 +368,58 @@ func DeleteIPsecEndpoint(remote *net.IPNet) {
 	ipsecDeleteXfrmPolicy(remote.IP)
 }
 
+func isXfrmPolicyCilium(policy netlink.XfrmPolicy) bool {
```

Comment:
@nebril looking at this particular commit backport, it seems like it just adds unused code. Did you have any other context around this or was it mistakenly marked for 1.6 backport? (/cc @jrfastab)

Reply:
It probably should be backported to 1.6 but is missing another patch that added DeleteXfrm(). @nebril I think the best thing to do here is drop the patch from the series get the rest of this 1.6 backports in then I'll do a backport series just for encryption. @joestringer sound ok to you?

Reply:
You are right, I missed that. Will drop it and leave the backport to @jrfastab

Reply:
@jrfastab will you also include #10268 ?